app/models/cohort_columns/last_homeless_visit.rb

```diff
@@ -27,9 +27,12 @@ def value(cohort_client, user) # OK
       # e.g.: {:project_name=>\"APR - Transitional Housing\", :date=>Mon, 30 Sep 2019, :project_id=>10}
       return unless lhv.present?
 
+      enforce_visibility = cohort_client.cohort.enforce_project_visibility_on_cells?
       lhv = JSON.parse(lhv)
       lhv.select do |row|
-        row['project_id'].in? user.visible_project_ids_enrollment_context
+        next true unless enforce_visibility
```

Comment:
Needs to return projects that are visible in the window if `cohort.only_window` otherwise true.